src/components/Balance.tsx

```diff
@@ -51,12 +53,13 @@ export default function Balance({ balance }: IBalanceProps) {
 				</Text>
 				<TouchableOpacity
 					style={styles.submitIssue}
-					onPress={() => void openUrl(repoIssueUrl)}
+					onPress={() => void openUrl(repoIssueUrl)?.catch((err: string) => openPromptAutoClose({ msg: err }) )}
```

Comment:
I think err is not a string.

catch(e => openPromptAutoClose({msg: e instanceof Error ? e.message : 'Link could not be opened'})